crates/api_models/src/payments.rs

```diff
@@ -1658,30 +1658,43 @@ pub struct PaymentListResponse {
     pub data: Vec<PaymentsResponse>,
 }
 
-#[derive(Clone, Debug, serde::Serialize, ToSchema)]
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct PaymentListFilterConstraints {
+    // The identifier for payment
+    pub payment_id: Option<String>,
```

Comment:
You can still have these as documentation comments, would help during development.